homeassistant/components/roborock/image.py

```diff
@@ -66,13 +66,7 @@ def __init__(
         self._attr_image_last_updated = dt_util.utcnow()
         self.map_flag = map_flag
         self.cached_map = self._create_image(starting_map)
-
-    @property
-    def entity_category(self) -> EntityCategory | None:
-        """Return diagnostic entity category for any non-selected maps."""
-        if not self.is_selected:
-            return EntityCategory.DIAGNOSTIC
-        return None
+        self._attr_entity_category = EntityCategory.DIAGNOSTIC
```

Comment:
WDYT of adding a separate (duplicate) non-diagnostic entity for the current map, so that something will always appear?

Reply:
I'm not sure if it is needed. It still always exist, it just doesn't auto get added to dashboard. User can always manually add it themselves.